promtimer/promtimer.py

```diff
@@ -252,7 +287,57 @@ def main():
     grafana_port = args.grafana_port
     prometheus_base_port = grafana_port + 1
 
-    if not args.cluster:
+    cbbackupmgr_stats_mode = False
```

Comment:
Suggest calling this: `backup_archive_mode` to be consistent with the name you gave to the command line parameter.